challenges/MeetPandya Coding Challenge

```diff
@@ -0,0 +1,46 @@
+#Back-end Coding challenge for Headstorm
+
+#Meet Pandya
+#email id : mnpandya10@gmail.com
+
+import flask
+import random
+from flask import jsonify, request
+
+app = flask.Flask(__name__)
```

Comment:
How did you decide to use Flask for this challenge? What do you like and not like about the Flask framework?

Reply:
Since I am a beginner at web development, I prefer to use Flask. Also in Flask, routing URLs is easy and it is extremely flexible. But what I do not like about Flask is that it has lack of database and limited features.